migrations/20170308164751_create_analytics_data.js

```diff
@@ -6,6 +6,7 @@ exports.up = function(knex) {
     table.dateTime("date_time")
     table.jsonb("data")
     table.timestamps(true, true)
+    table.string("version")
```

Comment:
this value comes back `null` if empty but keeps our tables in line

Reply:
Assuming this migration file is already applied to production, this change should be in a new migration file.

Reply:
I don't even think we need this value here at will. Will remove.